test/onnx/test_fx_op_consistency.py

```diff
@@ -1019,6 +1019,15 @@ def skip_torchlib_forward_compatibility(
         "pca_lowrank",
         reason=onnx_test_common.reason_dynamo_does_not_support("wrapper_set_seed"),
     ),
+    xfail(
+        "put",
+        dtypes=onnx_test_common.BOOL_TYPES,
+        reason=onnx_test_common.reason_onnx_script_does_not_support("index_put", "bool"),
+    ),
+    xfail(
+        "put",
+        reason=onnx_test_common.reason_dynamo_does_not_support("fixme: indices element out of data bound"),
+    ),
```

Comment:
These, can go now, and you should also remove `put` and `put.out` from `has_decomposition`, I believe.

Reply:
Removed.